chainerx_cc/chainerx/routines/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 add_library(chainerx_routines STATIC
+<<<<<<< HEAD
     arithmetic.cc
+=======
+    activation.cc
+>>>>>>> Add chainerx.elu
     binary.cc
```

Comment:
There is a syntax error in cmake here

Reply:
The build is failing due to an incorrect CMakeLists,
Please fix it :)

Reply:
Thanks @emcastillo !